### PR TITLE
Resolving issue #34 

### PR DESCRIPTION
**Pull request by ghost**
Resolving issue #34

### CODE REVIEW

**Comment by ghost** [comment]
I accidentally close this pull request and so reopen it just after that.

**Comment by yesnault** [comment]
The default Group must be identified by id 1, not by name.

**Comment by yesnault** [comment]
And the user have to know that he can't remove default group...

**Comment by ghost** [comment]
I think that it's better if a user can modify a group name into "DefaultGroup" in order to prevent some confusion. Otherwise it will be possible to have as many "DefaultGroup" as you wan't and users can delete one "DefaultGroup" and not other.
You are right users should be noticed when an edition or a deletion is refused, but i exactly how to produce a Scala exception and render a error message to the user after that.

File: app/models/Environment.scala [hunk 1]
```diff
@@ -205,31 +205,52 @@ object Environment {
   }
 
   /**
+   * Check that the given environment is unique for the corresponding group
```
Comment (yesnault) [inline app/models/Environment.scala:208]: An environment name have to be unique regardless the goup. Otherwise, the request on /soap/:group/ is ambiguous

**Comment by yesnault** [comment]
Simpler, see : https://github.com/soapower/soapower/commit/0200942e52d8cc493d4e8c1e3d807b37558f20d0

**Pull request closed without merging**